### PR TITLE
Added Staff Stuff

### PR DESCRIPTION
**Pull request by Meowkov**
Added:
crpg_disabled_snaplemask
crpg_disabled_landsknechthat_peacock
crpg_disabled_lemonfullmailcoif
crpg_disabled_yelmon
crpg_light_kuyak
crpg_disabled_strawhatgreathelm
crpg_disabled_strawlord_helmet
crpg_disabled_reinforcedottoman
crpg_splittingmaul
crpg_disabled_dragonscimitar
crpg_disabled_ibelinsword
crpg_disabled_dragonslayersword

### CODE REVIEW

File: itembalancing
Comment (Muparadzi) [inline itembalancing:1]: What is this? Should be deleted probably

Comment (Meowkov) [reply itembalancing:1]: removed, thanks :D

**Review by Erdertus — APPROVED** [review]

**Comment by Yeldur** [comment]
Conflicting currently -> Will address and then merge.

**Review by Yeldur — APPROVED** [review]
Approved

**Review by Muparadzi — APPROVED** [review]

**Pull request merged**